.gitattributes

```diff
@@ -1,4 +1,2 @@
-photon-api/src/integTest/resources/* linguist-vendored
-photon-api/src/test/resources/* linguist-vendored
-photon-client/src/integTest/resources/* linguist-vendored
-photon-client/src/test/resources/* linguist-vendored
+photon-ml/src/integTest/resources/* linguist-vendored
+photon-ml/src/test/resources/* linguist-vendored
```

Comment:
What's the purpose of this change?

Reply:
It was generated by Intelij. Will remove